photon-server/build.gradle

```diff
@@ -155,3 +156,31 @@ jacocoTestReport {
         }))
     }
 }
+
+
+remotes {
+    rpi {
+        host = 'photonvision.local'
+        user = 'pi'
+        password = 'raspberry'
+        knownHosts = allowAnyHosts
+    }
+}
+
+task deployToPi {
+    doLast {
+        println 'Starting deployment to Raspberry Pi photonvision.local'
+        ssh.run{
+            session(remotes.rpi) {
+                execute 'sudo systemctl stop photonvision.service'
+                execute 'sleep 3'
+                execute 'rm -f /opt/photonvision/photonvision.jar'
```

Comment:
Will this work without sudo?

Reply:
Summary from Discord conversation: No. 49b3d26ccf90bf12a02edbe51519089135314927 should fix this.